changelog/exceptions-opt.dd

```diff
@@ -0,0 +1,26 @@
+Exception trapping can now be disabled via `--DRT-extrap=1`
```

Comment:
That name seems a bit contradictory to the semantic. Doesn't `--DRT-extrap=0` make more sense ?